web/src/app/page.tsx

```diff
@@ -7,10 +8,11 @@ export const metadata = {
 
 export default function Home() {
   return (
-    <div className="page">
-      <h1 id="title--portal">Welcome to the ARC Services Portal</h1>
-      <p>This portal allows UCL researchers to manage ARC services and tasks.</p>
-
+    <div className="page page--home">
```

Comment:
The BEM structure is not entirely clear here, I was expecting to see a `page` block and a `title` block. Also, the `--home` could possibly be made more clear as it is not clear what the effects of the `--home` modification are. Ideally, your `--` modification naming should express what the modification does (e.g. centred, large etc.)